src/main/java/io/github/hapjava/accessories/AirPurifierAccessory.java

```diff
@@ -79,8 +79,23 @@
   /** Unsubscribes from changes in the target state of the air purifier. */
   void unsubscribeTargetState();
 
+  /**
+   * If a filter maintenance service is needed as a linked service to this AirPurifier, this is the
+   * place.
+   *
+   * @return an instance of FilterMaintenanceAccessory, null if not needed.
+   */
+  default FilterMaintenanceAccessory getFilterMaintenanceAccessory() {
+    return null;
+  };
+
   @Override
   default Collection<Service> getServices() {
-    return Collections.singleton(new AirPurifierService(this));
+    AirPurifierService service = new AirPurifierService(this);
```

Comment:
linked service approach is generic one, we dont need to change getService here. 
we can link any service with any other service, e.g. we can add filter service but also Fan or Slat

Reply:
I doubted on this one while adding it. But, I preserved the declarational properties of this lib. I mean, everything is done with templating / interfaces. Like the new feature #115(https://github.com/hap-java/HAP-Java/pull/115) this very generic characteristic is only possible when HAP-java allows it. That's not necessarily a bad thing, but a choice. It protects the user of HAP-java from adding the wrong stuff. It show's the limits of HAP in the interfaces. 

Leaving this out allong with getFilterMaintenanceAccessory() mean that you'll have to override getServices if you wan't to use it (not done by hab-java). That is the only alternative I could see.

What do you think?